terphenyl_simulations/build.py

```diff
@@ -0,0 +1,141 @@
+import yaml
+import os
+import sys
+import shutil
+from subprocess import Popen, PIPE
+import mbuild as mb
+import warnings
+from mbuild.lib.recipes.polymer import Polymer
+from .utils import ROOT_DIR, replace_all_pattern
+
+PACKMOL = os.path.abspath(os.path.join(ROOT_DIR, '../submodules/packmol/packmol'))
```

Comment:
This setup (bundling and building PACKMOL from within the package) could be worked around by adding it it as a dependency and [finding it](https://github.com/openforcefield/openff-interchange/blob/02058c27e32d764b6e27ec51dbf337a8ef3a5f23/openff/interchange/components/_packmol.py#L69-L85) through `$PATH`

(Of course if it works it works, just trying to offer an option for slimming things down where I see it)

Reply:
Hey Matt! Thanks for the input. It would be nice to not have to include PACKMOL as a submodule.

Is PACKMOL available from a package manager like mamba or conda-forge? I've only ever compiled it from source.

For the code you provided how is packmol installed on a clean environment?

Reply:
It's been on `conda-forge` for a while now: https://prefix.dev/channels/conda-forge/packages/packmol

but it's also often packaged within or alongside AmberTools so it might be accessible from there.

From a clean environment `mamba install packmol -c conda-forge` should work, or it should also be installable by listing it in the conda YAML like any other package (it's easy to forget that non-Python packages are treated just the same by this ecosystem). OpenFF's stack almost always has it installed from one source or another, so code I wrote can pretty much trust that it's there on $PATH, i.e.

```shell
$ micromamba repoquery -t whoneeds packmol
Using local repodata...

Loaded current active prefix: "/Users/mattthompson/micromamba/envs/interchange-examples-env"

 Name       Version Build             Depends     Channel Subdir
─────────────────────────────────────────────────────────────────
 ambertools 23.3    py311hb11241e_6   conda-forge osx-64
 mbuild     0.16.4  pyhd8ed1ab_0      conda-forge noarch
$ which packmol
/Users/mattthompson/micromamba/envs/interchange-examples-env/bin/packmol
```

So, not sure how you're handing dependencies here, but you can treat it as "just another" dependency alongside however else you're managing things

Reply:
Thanks for sharing this! I was looking on conda's webpages, but didn't check on conda-forge. I'm keeping all the dependencies in an environment.yml file, so I'll just add packmol to that!

Reply:
I finally moved over to mamba for my environment management and it makes such a big speed up